src/cascadia/TerminalCore/ITerminalApi.hpp

```diff
@@ -22,6 +23,7 @@ namespace Microsoft::Terminal::Core
         virtual TextAttribute GetTextAttributes() const noexcept = 0;
         virtual void SetTextAttributes(const TextAttribute& attrs) noexcept = 0;
 
+        virtual Microsoft::Console::Types::Viewport GetBufferSize() noexcept = 0;

```

Comment:
Is it OK to use `Microsoft::Console::Types::Viewport` here?

Reply:
The name `GetBufferSize` comes from the conhost code.

Reply:
I'd rather it be a `til::rectangle` as that's what I want to eventually replace all of `Viewport` usage... but if that means it has to convert in and out and is otherwise ugly... then I guess it's fine for now.

Reply:
Yeah ... it's going to be ugly if I switch to `til::rectangle`. I'll just leave this to future me.